index.html

```diff
@@ -11,28 +11,60 @@ <h1>My first Calculator</h1>
     <div class="magicWrapper"><button class="magicBtn">Press me!</button></div>
     <div class="calculator__wrapper">
       <div class="calculator">
-        <button class="clear">C</button>
+        <div class="magic__wrapper"><button class="clear">C</button></div>
         <div class="output__wrapper">
           <input type="text" class="calculator__output" />
         </div>
         <!-- button{$}*9 -->
 
-        <button class="calculator__inputs calculator__num">7</button>
-        <button class="calculator__inputs calculator__num">8</button>
-        <button class="calculator__inputs calculator__num">9</button>
-        <button class="calculator__input__spec">/</button>
-        <button class="calculator__inputs calculator__num">4</button>
-        <button class="calculator__inputs calculator__num">5</button>
-        <button class="calculator__inputs calculator__num">6</button>
-        <button class="calculator__input__spec">*</button>
-        <button class="calculator__inputs calculator__num">1</button>
-        <button class="calculator__inputs calculator__num">2</button>
-        <button class="calculator__inputs calculator__num">3</button>
-        <button class="calculator__input__spec">-</button>
-        <button class="calculator__inputs">.</button>
-        <button class="calculator__inputs calculator__num">0</button>
-        <button class="calculator__input__result">=</button>
-        <button class="calculator__input__spec">+</button>
+        <div class="magic__wrapper">
```

Comment:
I think it's not necessary to create a wrapper element. instead, you could apply the css of `magic__wrapper` to the buttons

Reply:
The buttons need to have `position: relative;` to move around. In this case i get issues with my flexbox around the buttons. My solution was, to put a container around every button, which is then related to the flexbox around. so the button in the end can be `position: relative;` and fly around, while their parent (magic__wrapper) will always on the final-position. So far i couldn't find another solution.